dashboard/app/controllers/home_controller.rb

```diff
@@ -34,6 +34,23 @@ def health_check
 
   GALLERY_PER_PAGE = 5
   def index
+    if request.cookies['pm'] == 'new_header'
```

Comment:
did we decide that there was no need to check if request.cookies exists because there will always be cookies, because that's how sessions are handled?